docs-web/Makefile

```diff
@@ -0,0 +1,28 @@
+NETLIFY_ROOT	=	/opt/build/repo
```

Comment:
could we possibly use the same assignment style as in the https://github.com/nginxinc/kubernetes-ingress/blob/master/Makefile ?
For example,
```
NETLIFY_ROOT = /opt/build/repo
```